test/integration/activate_int_test.go

```diff
@@ -46,7 +46,7 @@ func (suite *ActivateIntegrationTestSuite) TestActivateWithoutRuntime() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	cp := ts.Spawn("activate", "ActiveState-CLI/Python3")
+	cp := ts.Spawn("activate", "ActiveState-CLI/Python2")
```

Comment:
What's the purpose of changing these to Python 2?

Reply:
It is ugly. No debate here...

- The `state activate` command now always runs an activation event, which is interpreted by a python script.
- To speed these tests up, we are running with `ACTIVESTATE_CLI_DISABLE_RUNTIME=true`.  => We do not provide a `python` executable!  So, we rely on the `python` executable that is provided by the system.
- On Windows CI, we only have `python2` but not `python3`.

As the activation event is not really what we are testing here, I chose to do it this way.

The alternative but slower solution is to enable the runtime for these tests.